source/tools/glkernel-cli/helper.h

```diff
@@ -0,0 +1,127 @@
+#include <algorithm>
```

Comment:
This file only contains export funcitonality so far, maybe the name should be more specific

Reply:
Depends on what we want to do with it. Maybe a comment would help as well.